rca/project_styleguide/templates/patterns/organisms/programme_tabs/curriculum.html

```diff
@@ -14,4 +14,22 @@
     {% if page.what_you_will_cover_blocks %}
         {% include "patterns/organisms/accordion-block/accordion-block.html" with accordion_id='2' accordions=page.what_you_will_cover_blocks title='What you\'ll cover' modifier="accordion-block--last" %}
     {% endif %}
+
+    {% if page.working_with %}
+        <div class="section__row section__row--first-small-med-bp section__row--last">
+            <div class="section__header grid">
+                <h2 class="section__heading section__heading--secondary heading heading--five">{% firstof page.working_with_heading "The opportunity to work with…" %}</h2>
```

Comment:
Didn't realise you could use `firstof` like this - always though you'd do `{{ page.working_with_heading|default:"The opportunity to work with…" }}` with a fallback